README.md

```diff
@@ -279,6 +299,8 @@ arbitrary tombstoned subfeed under `metafeed` is returned.
 The response is delivered to the callback `cb`, where the 1st argument is the
 possible error, and the 2nd argument is the found tombstoned feed.
 
+-->
```

Comment:
Is this some leftover thing from commenting out stuff?

Reply:
Nope, that's a comment that is just hiding the deprecated APIs from the README, so that developers using this library try to use the other APIs for now. I didn't want to delete code, not yet

Reply:
Ah, nevermind now I see the start.